static/app/components/globalSdkUpdateAlert.tsx

```diff
@@ -116,9 +115,7 @@ function InnerGlobalSdkUpdateAlert(
   );
 }
 
-const WithSdkUpdatesGlobalSdkUpdateAlert = withSdkUpdates(
-  withPageFilters(InnerGlobalSdkUpdateAlert)
-);
+const WithSdkUpdatesGlobalSdkUpdateAlert = withSdkUpdates(InnerGlobalSdkUpdateAlert);
```

Comment:
It'd be nice if `withSdkUpdates` was converted to a custom hook, then we could get rid of the inner export.